docs/understanding-docker.md

```diff
@@ -17,274 +17,287 @@ weight = -90
 
-*Faster delivery of your applications*
+*Fast, consistent delivery of your applications*
 
-Docker is perfect for helping you with the development lifecycle. Docker
-allows your developers to develop on local containers that contain your
-applications and services. It can then integrate into a continuous integration and
-deployment workflow.
+Docker can streamline the development lifecycle by allowing developers to work in
+standardized environments using local containers which provide your applications
+and services. You can also integrate Docker into your continuous integration and
+continuous deployment (CI/CD) workflow.
 
-For example, your developers write code locally and share their development stack via
-Docker with their colleagues. When they are ready, they push their code and the
-stack they are developing onto a test environment and execute any required
-tests. From the testing environment, you can then push the Docker images into
-production and deploy your code.
+Consider the following example scenario. Your developers write code locally and
+share their work with their colleagues using Docker containers. They can use
+Docker to push their applications into a test environment and execute automated
+and manual tests. When problems are found, they can be fixed and the updated code
+can be deployed to the test environment. The tested Docker images can be pushed
+to the production environment to expose the new code to your customers.
 
-*Deploying and scaling more easily*
+*Responsive deployment and scaling*
 
 Docker's container-based platform allows for highly portable workloads. Docker
 containers can run on a developer's local host, on physical or virtual machines
-in a data center, or in the Cloud.
+in a data center, in the Cloud, or in a mixture of environments.
 
-Docker's portability and lightweight nature also make dynamically managing
-workloads easy. You can use Docker to quickly scale up or tear down applications
-and services. Docker's speed means that scaling can be near real time.
+Docker's portability and lightweight nature also make it easy to dynamically manage
+workloads, scaling up or tearing down applications and services as business
+needs dictate, in near real time.
 
-*Achieving higher density and running more workloads*
+*Running more workloads on the same hardware*
 
 Docker is lightweight and fast. It provides a viable, cost-effective alternative
-to hypervisor-based virtual machines. This is especially useful in high density
-environments: for example, building your own Cloud or Platform-as-a-Service. But
-it is also useful for small and medium deployments where you want to get more
-out of the resources you have.
+to hypervisor-based virtual machines, allowing you to use more of your compute
+capacity to achieve your business goals. This is useful in high density
+environments and for small and medium deployments where you need to do more with
+fewer resources.
 
 ## What is Docker's architecture?
```

Comment:
Maybe not for this PR, but this doc is _really_long. I think the information shifts toward the more technical here, and it would make a good place to break things out.